### PR TITLE
document required version for starting chain from scratch

### PR DESCRIPTION
**Pull request by galxy25**
https://app.shortcut.com/kava-labs/story/9537

### CODE REVIEW

File: docs/participate/faq/historic-data.mdx [hunk 1]
```diff
@@ -12,7 +12,8 @@ These endpoints are documented below with the relevant time frames for the data
 | Kava 7        | v0.14.2      | kava-7       | 1878508<sup>\*</sup> | 2021-04-08 | 2021-08-30 | [API](https://api.kava-7.archives.kava.io/node_info)<br/>[RPC](https://rpc.kava-7.archives.kava.io/status)               |
 | Kava 8        | v0.15.2      | kava-8       | 1803249<sup>\*</sup> | 2021-08-31 | 2022-01-19 | [API](https://api.kava-8.archives.kava.io/node_info)<br/>[RPC](https://rpc.kava-8.archives.kava.io/status)               |
 | Kava 9        | v0.16.1      | kava-9       | 1610470<sup>\*</sup> | 2022-01-19 | 2022-05-25 | [API](https://api.kava-9.archives.kava.io/node_info)<br/>[RPC](https://rpc.kava-9.archives.kava.io/status)               |
-| Kava 10       | v0.18.2      | kava_2222-10 | 2098400              | 2022-05-25 | 2022-10-26 | [API](https://api.data.kava.io/node_info)<br/>[RPC](https://rpc.data.kava.io/status)<br/>[EVM](https://evm.data.kava.io) |
+| Kava 10       | v0.17.7       | kava_2222-10 | 2098400              | 2022-05-25 | 2022-08-04 | [API](https://api.data.kava.io/node_info)<br/>[RPC](https://rpc.data.kava.io/status)<br/>[EVM](https://evm.data.kava.io) |
```
Comment (galxy25) [inline docs/participate/faq/historic-data.mdx]: wasn't sure what the correct height was for the kava 10 and 10.1 updates, doc https://kava-labs.atlassian.net/wiki/spaces/ENG/pages/1001979905/Accessing+Kava+Historical+Data has `974569` as the upgrade height for kava 10 but thats before the height of kava 9 so just defaulting to the same values until I get a more accurate value

Comment (rhuairahrighairidh) [reply docs/participate/faq/historic-data.mdx]: 974569 is correct for Kava 10, the chain was reset to 0 height for all launches before Kava 10.1. So the final heights are unconnected

Comment (rhuairahrighairidh) [reply docs/participate/faq/historic-data.mdx]: and 2098399 is the final height of Kava 10.1 (https://www.mintscan.io/kava/proposals/102 see height and subtract one).

Comment (galxy25) [reply docs/participate/faq/historic-data.mdx]: ahh now that makes sense - updated the heights accordingly

File: docs/participate/faq/historic-data.mdx [hunk 2]
```diff
@@ -12,7 +12,8 @@ These endpoints are documented below with the relevant time frames for the data
...
-| Kava 10       | v0.18.2      | kava_2222-10 | 2098400              | 2022-05-25 | 2022-10-26 | [API](https://api.data.kava.io/node_info)<br/>[RPC](https://rpc.data.kava.io/status)<br/>[EVM](https://evm.data.kava.io) |
+| Kava 10       | v0.17.7       | kava_2222-10 | 974569              | 2022-05-25 | 2022-08-04 | [API](https://api.data.kava.io/node_info)<br/>[RPC](https://rpc.data.kava.io/status)<br/>[EVM](https://evm.data.kava.io) |
+| Kava 10.1       | v0.18.1     | kava_2222-10 | 2098400              | 2022-08-04 | 2022-10-26 | [API](https://api.data.kava.io/node_info)<br/>[RPC](https://rpc.data.kava.io/status)<br/>[EVM](https://evm.data.kava.io) |
 | Kava 11       | v0.19.2      | kava_2222-10 | 3607200              | 2022-10-26 | 2023-02-26 | [API](https://api.data.kava.io/node_info)<br/>[RPC](https://rpc.data.kava.io/status)<br/>[EVM](https://evm.data.kava.io) |
 | Kava 12       | v0.21.1      | kava_2222-10 | 4832500              | 2023-02-26 | 2023-05-17 | [API](https://api.data.kava.io/node_info)<br/>[RPC](https://rpc.data.kava.io/status)<br/>[EVM](https://evm.data.kava.io) |
```
Comment (rhuairahrighairidh) [inline docs/participate/faq/historic-data.mdx]:
```suggestion
| Kava 11       | v0.19.2      | kava_2222-10 | 3607199              | 2022-10-26 | 2023-02-26 | [API](https://api.data.kava.io/node_info)<br/>[RPC](https://rpc.data.kava.io/status)<br/>[EVM](https://evm.data.kava.io) |
| Kava 12       | v0.21.1      | kava_2222-10 | 4832499              | 2023-02-26 | 2023-05-17 | [API](https://api.data.kava.io/node_info)<br/>[RPC](https://rpc.data.kava.io/status)<br/>[EVM](https://evm.data.kava.io) |
```
technically final blocks are one before upgrade heights
Kava 12 https://www.mintscan.io/kava/proposals/124
Kava 13 https://www.mintscan.io/kava/proposals/139

File: docs/participate/faq/historic-data.mdx [hunk 3]
```diff
@@ -12,9 +12,10 @@ These endpoints are documented below with the relevant time frames for the data
...
-| Kava 10       | v0.18.2      | kava_2222-10 | 2098400              | 2022-05-25 | 2022-10-26 | [API](https://api.data.kava.io/node_info)<br/>[RPC](https://rpc.data.kava.io/status)<br/>[EVM](https://evm.data.kava.io) |
-| Kava 11       | v0.19.2      | kava_2222-10 | 3607200              | 2022-10-26 | 2023-02-26 | [API](https://api.data.kava.io/node_info)<br/>[RPC](https://rpc.data.kava.io/status)<br/>[EVM](https://evm.data.kava.io) |
-| Kava 12       | v0.21.1      | kava_2222-10 | 4832500              | 2023-02-26 | 2023-05-17 | [API](https://api.data.kava.io/node_info)<br/>[RPC](https://rpc.data.kava.io/status)<br/>[EVM](https://evm.data.kava.io) |
+| Kava 10       | v0.17.7       | kava_2222-10 | 974569              | 2022-05-25 | 2022-08-04 | [API](https://api.data.kava.io/node_info)<br/>[RPC](https://rpc.data.kava.io/status)<br/>[EVM](https://evm.data.kava.io) |
+| Kava 10.1       | v0.18.1     | kava_2222-10 | 2098400              | 2022-08-04 | 2022-10-26 | [API](https://api.data.kava.io/node_info)<br/>[RPC](https://rpc.data.kava.io/status)<br/>[EVM](https://evm.data.kava.io) |
```
Comment (rhuairahrighairidh) [inline docs/participate/faq/historic-data.mdx]:
```suggestion
| Kava 10.1       | v0.18.1     | kava_2222-10 | 2098399              | 2022-08-04 | 2022-10-26 | [API](https://api.data.kava.io/node_info)<br/>[RPC](https://rpc.data.kava.io/status)<br/>[EVM](https://evm.data.kava.io) |
```

**Pull request merged**